pmm-app/src/pmm-dbaas/components/DBCluster/DBClusterName/DBClusterName.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC, useMemo } from 'react';
+import { Icon, useStyles } from '@grafana/ui';
+import { getStyles } from './DBClusterName.styles';
+import { DBClusterNameProps } from './DBClusterName.types';
+import { DASHBOARD_URL_MAP } from './DBClusterName.constants';
+
+export const DBClusterName: FC<DBClusterNameProps> = ({
+  dbCluster: {
+    clusterName,
+    databaseType,
+  },
+}) => {
+  const styles = useStyles(getStyles);
+  const dashboardURL = useMemo(() => (
```

Comment:
nit: while there is no harm, there is probably no benefit either, since this computation is not expensive. 
Quote: "This optimization helps to avoid expensive calculations on every render."
Reference: https://reactjs.org/docs/hooks-reference.html#usememo
One of other possible use cases of it is when we pass it to a child as a prop. Then it'll prevent the child from re-rendering.

Reply:
True, thanks! I'm working on PMM-6843 now so will remove it there.